test/app-tap/tarantoolctl.test.lua

```diff
@@ -327,11 +329,12 @@ do
 
     local status, err = pcall(function()
         test:test("check basic help", function(test_i)
-            test_i:plan(4)
+            test_i:plan(6)
             test_help(test_i, nil, "tarantoolctl", "Usage:")
             test_help(test_i, nil, "tarantoolctl help", "Usage:")
             test_help(test_i, nil, "tarantoolctl --help", "Usage:")
             test_help(test_i, dir, "tarantoolctl", "Usage:")
+            check_ok(test_i, dir, "rocks", "--help", 0, "tarantoolctl rocks %- Lua", nil)
```

Comment:
Nit: 'Lua' here is part of the following sentence:

> LuaRocks main command-line interface

First time I read the test, I was surprised: why Lua? I would do something like the following.

```lua
local exp = 'tarantoolctl rocks %- ' ..
    'LuaRocks main command%-line interface'
check_ok(test_i, dir, "rocks", "--help", 0, exp)
```

Reply:
fixed